src/main/java/com/redhat/cloud/notifications/models/Bundle.java

```diff
@@ -47,6 +49,14 @@
     @JsonIgnore
     private Set<Application> applications;
 
+    @OneToMany(mappedBy = "bundle", cascade = CascadeType.REMOVE)
+    @JsonIgnore
+    private Set<BehaviorGroup> behaviorGroups;
```

Comment:
I don't think we need this set. It would be HUGE if we tried to fetch it anyway. Or this has only the ones that are not tied to an specific account?

Reply:
I'm not sure we can get rid of the `Set` because it's part of a Hibernate relationship. However, it is lazy loaded and if we never force a fetch of this attribute, it should never be loaded and therefore not cause any problem.

Reply:
I'm worried if we accidentally use this. Could we just remove this variable? `BehaviorGroup` class already has the relation that we want (BehaviorGroup -> Bundle) anyway. If not, we need to be careful about it.

Reply:
That's a legitimate concern, I agree with you. I'll try removing it after I'm done fixing all broken APIs and tests.

Reply:
`@OneToMany` relation removed.

Reply:
Thanks!